sdk/inc/azure/core/az_platform.h

```diff
@@ -11,42 +11,14 @@
 
 #include <azure/core/_az_cfg_prefix.h>
 
-/**
- * @brief Gets the platform clock in milliseconds.
- * @remark The moment of time where clock starts is undefined, but if this function is getting
- * called twice with one second interval, the difference between the values returned should be equal
- * to 1000.
- * @return Platform clock in milliseconds.
- */
-AZ_NODISCARD int64_t az_platform_clock_msec();
+#if __has_include("az_platform_impl.h")
+    //Users never directly include az_impl
+    //......
+    #include <az_platform_impl.h>
+#else
+    // Spit out error
+    #error "Error go see the docs"
```

Comment:
Here like I commented elsewhere:
```suggestion
    #error "Error please see the docs"
```
with maybe a link to the specific doc